src/main/scala/zio/prelude/CommutativeBoth.scala

```diff
@@ -49,14 +55,18 @@ object CommutativeBoth extends LawfulF.Invariant[CommutativeBothDeriveEqualInvar
   implicit def ChunkCommutativeBoth: CommutativeBoth[Chunk] =
     new CommutativeBoth[Chunk] {
       def both[A, B](fa: => Chunk[A], fb: => Chunk[B]): Chunk[(A, B)] = fa zip fb
+
+      def bothPar[A, B](fa: => Chunk[A], fb: => Chunk[B]): Chunk[(A, B)] = fa zip fb
     }
 
   /**
    * The `CommutativeBoth` instance for `Exit`.
    */
-  implicit def ExitCommutativeBoth[E]: CommutativeBoth[({ type lambda[+a] = Exit[E, a] })#lambda] =
-    new CommutativeBoth[({ type lambda[+a] = Exit[E, a] })#lambda] {
+  implicit def ExitCommutativeBoth[E]: CommutativeBoth[Exit[E, +*]] =
+    new CommutativeBoth[Exit[E, +*]] {
```

Comment:
Unfortunately this won't work in Dotty, that's why we have to use the `lambda` trick, sad, I know :crying_cat_face: .
Or try it and then we'll see for sure.